src/screens/FeedScreen/RenderList.js

```diff
@@ -206,7 +206,9 @@ const styles = StyleSheet.create({
   footerWrapper: (h) => ({height: h, alignItems: 'center', justifyContent: 'center'}),
   contentReaction: (heightReaction) => ({
     maxHeight: heightReaction,
-    marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0
+    marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0,
+    borderTopWidth: 1,
+    borderTopColor: COLORS.light_silver
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here add a border to the top of the content reaction section. However, this border will always be present regardless of whether `heightReaction` is less than or equal to 0. If the intention was to only show the border when `heightReaction` is less than or equal to 0 (similar to the `marginBottom` style), then you should conditionally set these properties based on `heightReaction`. Here's how you can do it:

```diff
contentReaction: (heightReaction) => ({
  maxHeight: heightReaction,
  marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0,
+ borderTopWidth: heightReaction <= 0 ? 1 : 0,
+ borderTopColor: heightReaction <= 0 ? COLORS.light_silver : 'transparent'
}),
```
This way, the border will only appear when `heightReaction` is less than or equal to 0, and will be transparent otherwise.


<!-- This is an auto-generated comment by OSS CodeRabbit -->